android/package.json

```diff
@@ -15,13 +15,13 @@
 	],
 	"minSDKVersion": "16",
 	"vendorDependencies": {
-		"android sdk": ">=23 <=25.x",
-		"android build tools": ">=23 <=25.x",
-		"android platform tools": ">=17 <=25.x",
+		"android sdk": ">=25 <=26.x",
+		"android build tools": ">=25 <=26.x",
+		"android platform tools": ">=25 <=26.x",
```

Comment:
I don't think so, I've used `25.0.3` build tools before without issues. Not sure about the platform tools version though. I'll check that